sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -21,73 +21,100 @@ class Output(_InputOutputBase):
     """Define an output of a Component or Job.
 
     :param type: The type of the data output. Possible values include:
-                        'uri_folder', 'uri_file', 'mltable', 'mlflow_model', 'custom_model', and user-defined types.
+        'uri_folder', 'uri_file', 'mltable', 'mlflow_model', 'custom_model', and user-defined types.
+        Defaults to 'uri_folder'.
     :type type: str
-    :param path: The path to which the output is pointing. Needs to point to a cloud path.
-    :type path: str
+    :param path: The path to which the output is pointing. Needs to point to a cloud path. Defaults to None.
+    :type path: str, optional
     :param mode: The mode of the data output. Possible values are:
-                        'rw_mount': Read-write mount the data,
-                        'upload': Upload the data from the compute target,
-                        'direct': Pass in the URI as a string
-    :type mode: str
-    :param description: Description of the output
-    :type description: str
-    :param name: The name used to register output as data or model asset. Name can be set without setting version.
-    :type name: str
+        'rw_mount': Read-write mount the data,
+        'upload': Upload the data from the compute target,
+        'direct': Pass in the URI as a string.
+        Defaults to None.
+    :type mode: str, optional
+    :param description: Description of the output. Defaults to None.
+    :type description: str, optional
+    :param name: The name used to register output as data or model asset.
+        Name can be set without setting version.
+        Defaults to None.
+    :type name: str, optional
     :param version: The version used to register output as data or model asset.
         Version can be set only when name is set.
-    :type version: str
+        Defaults to None.
+    :type version: str, optional
+    :param is_control: Determine if the output is a control output. Defaults to None.
+    :type is_control: bool, optional
+    :param early_available: Mark the output for early node orchestration. Defaults to None.
+    :type early_available: bool, optional
+    :param intellectual_property: Intellectual property associated with the output.
+        It can be an instance of `IntellectualProperty` or a dictionary that will be used to create an instance.
+        Defaults to None.
+    :type intellectual_property: Union[
+                                ~azure.ai.ml.entities._assets.intellectual_property.IntellectualProperty,
```

Comment:
Indent too long?

Reply:
If the length will not be too long after wrapping, then align with the previous line.
Otherwise, if it is still too long after the line break, then reduce the indentation.
I will modify this.